p4-16/psa/PSA.mdk

```diff
@@ -472,20 +471,12 @@ assist P4 developers in debugging their code.
 
     psa_ingress_output_metadata_t ostd;
 
-    if (ostd.class_of_service value is not supported) {
-        ostd.class_of_service = 0;    // use default class 0 instead
-        // Recommended to log error about unsupported
-        // ostd.class_of_service value.
-    }
```

Comment:
The block of pseudocode above was not deleted, just moved a little bit later, so it was closer to the rest of the non-clone packet handling.